doc/source/how-to/contributing.rst

```diff
@@ -0,0 +1,250 @@
+* ``Pull requests``: a board collecting new changes to be merged.
+* ``Discussions``: dialogs about development practices to be used by project maintainers.
+* ``Actions``: the CI/CD panel showing the different available workflows.
+* ``Projects``: a panel for organizing and planning development in the repository.
+* ``Wiki``: devoted to show quick information of the project.
+* ``Security``: configurations related to security issues, vulneravilities and alerts.
+* ``Insights``: generic information about the repository and its contributors.
+* ``Settings``: allows to configure access and integration with third party tools.
+
+.. note:: 
+
+   Depending on your access level, you may not be able to see some of the
+   previous sections.
+
+Creating an Issue
+-----------------
+Issues are the way of requesting for help, a new feature or reporting a bug.
+They act like small posts including a comments section. This last section allows
+you to interact with other users, developers and project maintainers.
+
+To open an issue, check the ``Issues`` section in the :ref:`GitHub Repository
+Sections` and click ``New Issue``. GitHub issues require the usage of Markdown
+instead of RST. You can check `Basic Writing and Formatting Syntax
+<https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax>`_
+for this language.
+
+Requesting New Features
+~~~~~~~~~~~~~~~~~~~~~~~
+If you would like a new feature to be included in the software, you can create a
+new issue. When requesting a new feature, it is interesting that you include the
+following sections:
+
+- Describe the main goal of the feature to be included and why it is bennefitial
+  to the project.
+
+- If possible, describe how it could get implemented and which steps should be
+  followed.
+
+- Add any references which could help during the development process.
+
+
+Reporting Bugs
+~~~~~~~~~~~~~~
+If you encounter some bug in your code, you can create a new issue to request a
+fix. When reporting a bug try to include the following data:
+
+- Indicate your operating system, Python version and the version of the library
+  you are using.
+
+- Include a small piece of code to allow others to reproduce the bug you found.
+
+- Any additional data you consider useful for fixing the bug.
+
+
+Forking a Repository
+--------------------
+Forking a repository is like copy-pasting a project into your own GitHub
+profile. Notice that only ``public`` labeled repositories can be forked. Those
+labeled as ``internal`` or ``private`` cannot be forked.
+
+To fork a repository click in the ``Fork`` button at the top of the project's
+page you wish to fork.
+
+
+Cloning a Repository
+--------------------
+Cloning a repo means downloading it to your local machine. There two ways of
+doing this: using ``HTTPS`` or ``SSH``. To force the usage of ``SSH``, only this
+method is exaplined in here.
+
+Cloning using SSH
```

Comment:
I don't know whether to change it to "Using" for now, until we move to sentence case...?